libs/sdk-backend-tiger/src/backend/workspace/attributes/elements/index.ts

```diff
@@ -167,15 +177,33 @@ class TigerWorkspaceElementsQuery implements IElementsQuery {
                     return client.labelElements.computeLabelElementsPost(elementsRequestWrapped);
                 });
 
-                const { paging, elements } = response.data;
+                const { paging, elements, format, granularity } = response.data;
+
+                const elementsGranularity = granularity as ElementsResponseGranularityEnum;
+                const sdkGranularity = toSdkGranularity(elementsGranularity);
+                const locale = format?.locale as FormattingLocale;
+                const pattern = format?.pattern as string;
+                const dateValueFormatter = createDateValueFormatter(this.dateFormatter);
 
                 return {
-                    items: elements.map(
-                        (element): IAttributeElement => ({
+                    items: elements.map((element): IAttributeElement => {
+                        const objWithFormattedTitle =
+                            sdkGranularity && format
```

Comment:
Maybe extract this to a property above the map to something like `shouldFormatTitle`